index.js

```diff
@@ -30,159 +12,72 @@ async function getIssues() {
   return issues;
 }
 
-async function createLabel() {
-  if (!labelName) {
+async function getOrCreateLabel(linearClient, label) {
```

Comment:
We'll be moving stories in both Engineering and Backend teams, so the label will be created in a run for one team and then only retrieved in the other.